src/components/Buttons/Button.tsx

```diff
@@ -34,11 +34,11 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
         ref={ref}
         {...rest}
         className={clsxm(
-          'inline-flex items-center rounded-full',
```

Comment:
don't change button style, instead where button is used, just pass there classname,